packages/core/src/fhir-normalization/normalize-fhir.ts

```diff
@@ -0,0 +1,36 @@
+import { Bundle, Resource } from "@medplum/fhirtypes";
+import { buildBundleEntry, extractFhirTypesFromBundle } from "../external/fhir/shared/bundle";
+import { cloneDeep } from "lodash";
+import { capture } from "../util";
+import { normalizeObservations } from "./resources/observation";
+
+export function normalizeFhir(
+  fhirBundle: Bundle<Resource>,
+  cxId: string,
+  patientId: string
+): Bundle<Resource> {
+  const normalizedBundle: Bundle = cloneDeep(fhirBundle);
```

Comment:
nit: cloneDeep already returns w/ type, so no need to put that on the `const`, right? See how its slightly different than the one on the param and return types